### ISSUE TITLE
Where to get a built version of the game?

### ISSUE THREAD

**Issue by LostLuuv** [xtreme8000/CavEX#62]
the process for building the game seems really complicated and a little daunting, is there anywhere i can get a built wii version?

**Comment by Caveproject** [comment]
Download it here https://oscwii.org/library/app/cavex, put it in your SD Card/USB, and open up homebrew and the game should appear.

**Comment by LostLuuv** [comment]
tysm!

**Comment by LostLuuv** [comment]
oh it seems like this is a very old build of the game

**Comment by xtreme8000** [comment]
You can find the original files [here](https://aos.party/download_public/). I might put them up as GitHub releases (with free assets however to avoid copyright).

**Comment by LostLuuv** [comment]
are you sure that that is an updated version? im not seeing any of the recent commits in-game

**Comment by Caveproject** [comment]
Did you still find the updated version?

**Comment by LostLuuv** [comment]
no, the versions linked were last updated december of last year

**Comment by LostLuuv** [comment]
any further info on where i could get an updated build of the game? if not i think i should close this issue

**Comment by Caveproject** [comment]
I don't think there are any updated version currently. But I hope Xtreme8000 will work on getting new updates on this game .

**Comment by joselemk** [comment]
There are built versions of the game online, however none of them seem to have the door placement fix, so I had to build my own.
If anyone wants to save the hassle of building this, I will leave a google drive link below.
[Google Drive Link](https://drive.google.com/drive/folders/1k9LIwt1RhS1jCJk7uK7a0RrYmnN4r-0m?usp=drive_link)
